src/accelerate/components/Hero.tsx

```diff
@@ -44,10 +53,18 @@ export default function Hero( props: Props ) {
 					{ !! props.period && ( props.canViewAnalytics || props.canViewInsights ) && (
 						<nav className='Hero__links'>
 							{ props.canViewAnalytics && (
-								<a href="index.php?page=altis-analytics">{ __( 'Analytics', 'altis-analytics' ) }</a>
+								<Link
+									url="index.php?page=altis-analytics"
+									location="dashboard"
+									linkText="Analytics"
```

Comment:
This is where the actual text for link is, so this where the translation functions can be used:
```suggestion
									linkText={ __( 'Analytics', 'altis-analytics' ) }
```